test/binary/suite.ini

```diff
@@ -2,4 +2,4 @@
 core = tarantool
 script = binary.lua
 description = memcached binary tests
-disabled = binary-boundary.test.py
+disabled = binary-expire.test.py binary-boundary.test.py
```

Comment:
I guess you had an idea to disable it and return here later, but then you found a fix: the next commit enables the test back. So, it seems, the commit that disables the test is not needed anymore?

Reply:
Right, forgot to remove it. Thanks!